projects/bootloader/protos/querying_bootloader.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+message QueryingMCU {
```

Comment:
What's the `MCU` suffix?

Also I don't think you need the `_bootloader` suffix in this file and querying_response_bootloader.proto.

Reply:
oh that was originally to distinguish between the response and client, but i figured out a better way to do it, some im just removing that